### PR TITLE
Expose the newly added statistics to perf_client output

### PR DESCRIPTION
**Pull request by tanmayv25**

### CODE REVIEW

File: src/clients/c++/perf_client/perf_client.cc [hunk 1]
```diff
@@ -1303,9 +1304,20 @@ main(int argc, char** argv)
 
       for (PerfStatus& status : summary) {
         uint64_t avg_queue_ns = status.server_stats.queue_time_ns /
-                                status.server_stats.request_count;
-        uint64_t avg_compute_ns = status.server_stats.compute_time_ns /
-                                  status.server_stats.request_count;
+                                status.server_stats.execution_count;
```
Comment (deadeyegoodwin) [inline src/clients/c++/perf_client/perf_client.cc:1307]: Have to be careful with counts. The success count tells you how many successful requests there were. inference_count tells you how many inferences were performed where a batch size n request is counted as n inferences. execution_count tells you how many times the model ran... which with dynamic batching can be significantly less than the number of requests or inference_count. I'm not sure you are using execution_count correctly here and maybe elsewhere.

File: src/clients/c++/perf_client/triton_client_wrapper.cc [hunk 2]
```diff
@@ -286,12 +286,15 @@ TritonClientWrapper::ParseStatistics(
                       std::make_pair(this_stat.name(), this_stat.version()),
                       ModelStatistics())
                   .first;
-    it->second.request_count_ = this_stat.inference_stats().success().count();
```
Comment (deadeyegoodwin) [inline src/clients/c++/perf_client/triton_client_wrapper.cc:289]: I think you still want request_count

File: src/clients/c++/perf_client/perf_client.cc (hunk 1 above)
Comment (tanmayv25) [reply src/clients/c++/perf_client/perf_client.cc:1307]: Is it safe to assume the success metric count to be same as other metric counts(compute_*, queue*) as we were doing earlier? Or should we just record the average using there own individual metric counts?

Comment (tanmayv25) [reply src/clients/c++/perf_client/perf_client.cc]: I was wrong assuming the execution count to be equal to the number of times these metrics were added. Fixed.

**Pull request merged**